docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -1243,6 +1243,22 @@ window.document.addEventListener('DOMContentLoaded', () => {
   );
 });
 ```
+[hint]
+GraphQL scaffolding derives the names for the schema from the first part of the namespace and the classname, while in
+the Injector the database table name is used. So in the case of
+```php
+namespace Foo\Bar;
+// …
+class MyVersionedObject extends DataObject
+{
+    private static $table_name = 'FBVersionedObject';
```

Comment:
Would `FooBarMyVersionedObject` be a better table name (or at least FBMyVersionedObject) ?  Little bit weird not including 'My' in the table_name when it's in the class name

Reply:
```suggestion
    private static $table_name = 'MyTableName';
```
I think this would help communicate it a little clearer